### ISSUE TITLE
Review annotations to GO:0006182 cGMP biosynthetic process

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#1977]
* GO:0006182 cGMP biosynthetic process
https://docs.google.com/spreadsheets/d/1W0vSSm4M6zB--coMKoYKMRn-bfBU4GXwEzCWhruWhKk/edit#gid=0

Group | Annotation count
-- | --
dictyBase | 2 
FlyBase | 8
GeneDB | 2
MGI | 7
MTBBASE | 1
RGD | 13
TAIR | 1
UniProt | 17
WB | 3

**Comment by tberardini** [comment]
TAIR annotation removed.

**Comment by vanaukenk** [comment]
WB annotations removed.

@pgaudet - just to confirm, are you going to remove the F-P link between guanylate cyclase activity (GO:0004383)  and cGMP biosynthetic process (GO:0006182), or just obsolete cGMP biosynthetic process (GO:0006182)?

I just want to make sure the BP annotations don't sneak back in due to the F-P link.  Thx.

**Comment by ukemi** [comment]
MGI's manual annotations have been removed.

**Comment by vanaukenk** [comment]
@pgaudet 
Yes, I think that's okay.  I just wanted to be clear on exactly what the plan was.  Thx.

**Comment by pgaudet** [comment]
Excellent. 

Techically the documents can now be edited, sorry about that.


Pascale

**Comment by hattrill** [comment]
FlyBase - done

**Comment by pfey03** [comment]
Done and replaced

**Comment by slaulederkind** [comment]
RGD annotations changed or deleted.

**Comment by pgarmiri** [comment]
The 8 EBI UniProt annotations have been reviewed and changed.
@sylvainpoux , could you please ask the curators at SIB to review their annotations? The names of the curators whose annotations are affected are in the spreadsheet.
Thanks

**Comment by suzialeksander** [comment]
everything on the spreadsheet was addressed, with one disputed in p2go
